packages/block-library/src/columns/transforms.js

```diff
@@ -8,8 +8,55 @@ import {
 
 const MAXIMUM_SELECTED_BLOCKS = 6;
 
+function fromMediaText( attributes, innerBlocks ) {
```

Comment:
I don't see a reason for this function to be here. I think it's better for readability to include it in the `transform` as it will not be reused.

Reply:
I've inlined it in the latest commit. It was mostly to save some indentation in order to have slightly more compact formatting. Plus, if there were to be more transformations I think it makes the `transforms` declaration more scannable.